charts/opentelemetry-collector/templates/_pod.tpl

```diff
@@ -6,6 +6,12 @@ imagePullSecrets:
 serviceAccountName: {{ include "opentelemetry-collector.serviceAccountName" . }}
 securityContext:
   {{- toYaml .Values.podSecurityContext | nindent 2 }}
+{{- if .Values.hostAliases }}
```

Comment:
`with` treats `[]` as false, so we can exclude the `if` statement

Reply:
Hi @TylerHelmuth,
I removed it.
Thx